README.md

```diff
@@ -147,11 +174,11 @@ Then in a template:
 This object knows its own prefix, by the way, so it wouldn't be necessary to
 use `prefix=` for disambiguation.
 
-Make it fancier with a mask:
```

Comment:
Was this removed on purpose?

Reply:
I removed it since the mask example code can be found in [Features > Advanced > Masking](https://github.com/scottwernervt/ember-fontawesome/tree/readme-toc#advanced) which matches the format of FortAwesome/vue-fontawesome@3b1b483. Do you want it back?

Reply:
Ah no that's mo' betta. Thank you!